rest-api-model-server/src/test/kotlin/ReplicatedRepositoryInstantiationTest.kt

```diff
@@ -0,0 +1,28 @@
+package org.acme.getting.started.testing
+
+import io.quarkus.test.junit.QuarkusTest
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Test
+import org.modelix.model.client.ReplicatedRepository
+import javax.inject.Inject
+
+@QuarkusTest
+class ReplicatedRepositoryInstantiationTest {
+    @Inject
+    var replicatedRepository: ReplicatedRepository? = null
```

Comment:
We've usuallu used a lateinit var for these cases. Constructor injection could also work here

Reply:
Done.
Used `lateinit`.
Constructor injection does not work. See. https://github.com/quarkusio/quarkus/issues/34330